guide/creating-your-bot/creating-commands.md

```diff
@@ -81,96 +95,177 @@ const rest = new REST({ version: '9' }).setToken(token);
+```sh:no-line-numbers
+node index.js
+```
+and then use the `/ping` command in your guild. You should see something logged in your terminal that is similar to this:
+```:no-line-numbers
+CommandInteraction {
+  type: 'APPLICATION_COMMAND',
+  id: '123',
+  applicationId: '123',
+  channelId: '123',
+  guildId: '123',
+  user: User { ... },
+  member: GuildMember { ... },
+  version: 1,
+  commandId: '123',
+  commandName: 'ping',
+  deferred: false,
+  options: CommandInteractionOptionResolver {
+    _group: null,
+    _subcommand: null,
+    _hoistedOptions: []
+  },
+  replied: false,
+  ephemeral: null,
+  webhook: InteractionWebhook { id: '123' }
+}
+```
+
+If you see something like that getting logged in your terminal, then that means your bot is receiving the interaction event correctly! However, our bot hasn't replied to the interaction yet, which is why you will see something like this in your Discord:
 
-### Server info command
+![](./images/interactionfailed.png)
 
-Note that servers are referred to as "guilds" in the Discord API and discord.js library. `interaction.guild` refers to the guild the interaction was sent in (a <DocsLink path="class/Guild" /> instance), which exposes properties such as `.name` or `.memberCount`.
+Let's go ahead and fix that by making these changes inside the callback:
+
+:::: code-group
+::: code-group-item index.js
+```js {14-22}
+// Require the necessary discord.js classes
+const { Client, Intents } = require('discord.js');
+const { token } = require('./config.json');
+
+// Create a new client instance
+const client = new Client({ intents: [Intents.FLAGS.GUILDS] });
+
+// When the client is ready, run this code (only once)
+client.once('ready', () => {
+	console.log('Ready!');
+});
 
-```js {9}
 client.on('interactionCreate', async interaction => {
 	if (!interaction.isCommand()) return;
 
-	const { commandName } = interaction;
-
-	if (commandName === 'ping') {
+	if (interaction.commandName === 'ping') {
 		await interaction.reply('Pong!');
-	} else if (commandName === 'server') {
+	} else if (interaction.commandName === 'server') {
 		await interaction.reply(`Server name: ${interaction.guild.name}\nTotal members: ${interaction.guild.memberCount}`);
-	} else if (commandName === 'user') {
-		await interaction.reply('User info.');
+	} else if (interaction.commandName === 'user') {
+		await interaction.reply(`Your tag: ${interaction.user.tag}\nYour id: ${interaction.user.id}`);
 	}
 });
+
+// Login to Discord with your client's token
+client.login(token);
 ```
+:::
+::::
+
+As we mentioned earlier, a Slash Command is just one type of interaction. Therefore, we should first check if the interaction is a Slash Command by calling `.isCommand()`. Next, we check the name of the command by accessing the `.commandName` property. Finally, we reply to the interaction by calling `.reply()`.
+
+::: danger
+You MUST reply to the interaction within **3 seconds** of receiving it.
+:::
```

Comment:
This should rather be a warning block, the capitalizations seems out of place as well and the number should be written out.

Reply:
👍🏻 i will make these changes